### DISCUSSION TITLE
HTTP 500 after logging into classroom dashboard

### DISCUSSION THREAD

**Discussion by Lolash** [community/community#141760] [GitHub Education]
### Hi everyone,

I cannot access GitHub Classroom at all. I cannot do it via assignment invitation nor through classroom dashboard. I tried doing it in browser incognito mode in two different browsers - Firefox and Chrome. 
In both cases everything I see is HTTP 500 error screen. 

GitHub Team - could you please take a look?
Thanks!

**Comment by jonjoncardoso** [comment]
I am having the exact same issue. Very frustrating

**Reply by Lolash** [reply]
Hi, the only workaround I found is to create a new github account and use it to access classroom.
